pimd/pim_cmd.c

```diff
@@ -575,7 +575,7 @@ static void igmp_show_interfaces_single(struct pim_instance *pim,
 					struct vty *vty, const char *ifname,
 					bool uj)
 {
-	struct igmp_sock *igmp;
+	struct gm_sock *igmp;
```

Comment:
Same comment as (https://github.com/FRRouting/frr/pull/10167). We can rename the function, file & variable name as well with gm prefix instead of igmp.

Reply:
Thanks Rathna for reviewing it. We have not changed the function names yet because it is still doing igmp work. While we will implement MLD, we will change the function names if required. Few things are different between igmp and mld. We are planning to have separate file for MLD and IGMP. So whatever will be common will be renamed in terms of function when we raise the PR for MLD.